nvim/lua/plugins/plugins_list.lua

```diff
@@ -96,6 +96,9 @@ local markdown_preview = {
     build = function() vim.fn["mkdp#util#install"]() end,
 };
 
+-- Add nvim-nio
```

Comment:
this comment does not really say much, I try to have one-line description of the plugin, e.g.
```lua
-- asynchronous IO library
```